tasks/github_tasks.py

```diff
@@ -165,3 +167,19 @@ def get_milestone_id(_, milestone):
     if not m:
         raise Exit(f'Milestone {milestone} wasn\'t found in the repo', code=1)
     print(m.number)
+
+
+@task
+def send_rate_limit_info_datadog(_, pipeline_id):
+    from .libs.common.github_api import GithubAPI
+
+    gh = GithubAPI('DataDog/datadog-agent')
+    rate_limit_info = gh.get_rate_limit_info()
+    print(f"Remaining rate limit: {rate_limit_info[0]}/{rate_limit_info[1]}")
+    metric = create_count(
+        metric_name='github.rate_limit.remaining',
```

Comment:
Isn't this the number of request we already used?
ie. shouldn't the remaining number of requests be `rate_limit_info[1] - rate_limit_info[0]` ?

Reply:
`get_rate_limit()` returns a tuple with `(requests remaining, request limit)` . So I think this what we want

Reply:
Then all good :)